app/views/ParentsViewComponent.vue

```diff
@@ -0,0 +1,244 @@
+        img.img-responsive(src="/images/pages/parents/valueprop1.png")
+        h4(data-i18n="parent_landing.value_head1")
+        p(data-i18n="parent_landing.value_copy1")
+      .col-lg-4.col-md-4
+        img.img-responsive(src="/images/pages/parents/valueprop2.png")
+        h4(data-i18n="parent_landing.value_head2")
+        p(data-i18n="parent_landing.value_copy2")
+      .col-lg-4.col-md-4
+        img.img-responsive(src="/images/pages/parents/valueprop3.png")
+        h4(data-i18n="parent_landing.value_head3")
+        p(data-i18n="parent_landing.value_copy3")
+    .row.text-center
+      .refer-btn
+        button.btn.btn-forest.btn-lg(v-on:click="$emit('referTeacher')", data-i18n="parent_landing.refer_teacher")
+
+  #deep-dive.container
+    .row.text-center
+      #dive-graphic-1.col-lg-5.col-md-12.col-sm-12
+        div
+        h2(data-i18n="parent_landing.dive_head1")
+      
+      .col-lg-6.col-md-12.col-sm-12.col-sm-offset-1.flags
+        .row
+          img(src="/images/pages/parents/flag.png").col-lg-4.flag-img
+          .col-lg-8.col-md-12.col-sm-12
+            h5(data-i18n="parent_landing.medical_flag")
+            p.small(data-i18n="parent_landing.medical_flag_copy")
+        .row
+          img(src="/images/pages/parents/flag.png").col-lg-4.flag-img
+          .col-lg-8.col-md-12.col-sm-12
+            h5(data-i18n="parent_landing.explore_flag")
+            p.small(data-i18n="parent_landing.explore_flag_copy")
+        .row
+          img(src="/images/pages/parents/flag.png").col-lg-4.flag-img
+          .col-lg-8.col-md-12.col-sm-12
+            h5(data-i18n="parent_landing.filmaking_flag")
+            p.small(data-i18n="parent_landing.filmaking_flag_copy")
+  
+    .row
+      #dive-graphic-2.col-lg-6.col-md-12.col-sm-12.col-lg-push-6.text-center
+        div
+        h2(data-i18n="parent_landing.dive_head2")
+      .col-lg-5.col-md-12.col-sm-12.col-lg-pull-6
+        p
+          span.spr(data-i18n="parent_landing.dive_par1")
+          a(href="https://www.learntechlib.org/p/19982/", data-i18n="parent_landing.dive_link1")
+          span.spl.spr(data-i18n="parent_landing.dive_par2")
+          a(href="http://webcache.googleusercontent.com/search?q=cache:v-UZ4U5_b7cJ:www.ejmste.com/A-Study-of-the-Effects-of-Digital-Learning-on-Learning-Motivation-and-Learning-Outcome,69635,0,2.html+&amp;cd=1&amp;hl=en&amp;ct=clnk&amp;gl=us", data-i18n="parent_landing.dive_link2")
+          span.spl.spr(data-i18n="parent_landing.dive_par3")
+          a(href="https://www.healio.com/nursing/journals/jcen/2016-8-47-8/%7B4c67edb9-1a9c-432d-af17-0f2856f3ce2c%7D/importance-of-gamification-in-increasing-learning", data-i18n="parent_landing.dive_link3")
+          span.spr(data-i18n="parent_landing.dive_par4")
+          a(href="http://www.sun.ac.za/english/learning-teaching/ctl/Documents/Gamification%20in%20education.pdf", data-i18n="parent_landing.dive_link4")
+          span.spl(data-i18n="parent_landing.dive_par5")
```

Comment:
Fyi, you can parameterize i18n strings:
https://github.com/codecombat/codecombat/blob/dc99e76/app/templates/core/create-account-modal/setup-account-panel.jade#L25
https://github.com/codecombat/codecombat/blob/dc99e76/app/locale/en.coffee#L304

Can sometimes avoid breaking up paragraphs like this.

Reply:
Will revisit this when Sophia and Robin have done more copy changes.